client/webserver/site/src/html/wallets.tmpl

```diff
@@ -88,6 +89,8 @@
   {{- /* RIGHT SIDE */ -}}
   <div id="rightBox" class="col-10 px-2 py-5 d-flex justify-content-center align-items-start overflow-auto">
 
+    <div class="fs14 pt-3 text-center d-hide errcolor" id="rightBoxErr"></div>
```

Comment:
The problem with having a single error message box outside of the forms is that this will need to be cleared for any new action the user does anywhere on the wallets page, otherwise the error message will persist forever. My read is that you need to do this because unlocking a wallet with a cached password no longer requires display of any forms. I'm thinking the best solution then is to just display the unlock form but hide the app password field, showing only the error message.